src/server/dragonfly_test.cc

```diff
@@ -398,7 +402,8 @@ TEST_F(DflyEngineTest, Eval) {
 
   // a,b important here to spawn multiple shards.
   resp = Run({"eval", "return redis.call('exists', KEYS[2])", "2", "a", "b"});
-  EXPECT_EQ(2, GetDebugInfo().shards_count);
+  // TODO: We have no way of measuring now.
+  // EXPECT_EQ(2, GetDebugInfo().shards_count);
```

Comment:
The issue is:
- this info was set after InitByArgs inside dispatch
- We deduce the number of active shards only in the handler

So we probably need to update the debug info inside the handler itself?